lib/main/lang/fixmyts/quickFixes/addImportFromStatement.ts

```diff
@@ -9,76 +9,74 @@ import {getPathCompletions} from "../../modules/getPathCompletions";
 
 function getIdentifierAndFileNames(error: ts.Diagnostic, project: Project) {
 
-    var errorText: string = <any>error.messageText;
+  var errorText: string = <any>error.messageText;
```

Comment:
Please don't change formatting if you're only changing a few lines in the file. Here, for example, I can't tell for sure if you've added or removed anything.